README.md

```diff
@@ -1,32 +1,71 @@
-# Intro
+# Storyboard Controller
 
-## Problem at hand
+`StoryboardController` provides you with a mechanism to inject dependencies into view controllers that are initialised
+from the storyboard which the controller monitors.
+
+## Why is it useful?
 
 In order to inject dependencies into view controllers, it is common to use one of the following patterns:
 
 * `prepare(for segue: UIStoryboardSegue, sender: Any?)` in a parent view controller (with storyboard)
 * custom initalisers (no storyboard)
 
-**However**, the above solutions do not clearly isolate each view controllers. As each of the view controllers
+**However**, the above solutions do not clearly isolate each view controller as each of the them
 will have to understand how the next one works in order to inject their dependencies.
 
-You then end up with a dependency graph that will look like this:
+You end up with a dependency graph that looks like this:
 
     ViewControllerA <- ViewControllerB <- ViewControllerC <- etc.
 
-It is then very hard to launch only `ViewControllerC` from somewhere else in the application
-as it obtains its dependencies from `ViewControllerB` which also needs `ViewControllerA` for its own dependencies.
-
-## Proposed Solution
-
-`StoryboardController` provides you with a mechanism to inject dependencies into view controllers that are initialised
-from the storyboard which the controller monitors.
+This makes it very hard to launch only `ViewControllerC` from somewhere else in the application
+as it obtains its dependencies from `ViewControllerB` which in turn obtains its dependencies from `ViewControllerA`.
 
-The dependency graph then looks like this:
+When using the `StoryboardController`, your dependency graph would now look like this:
 
     StoryboardController <- ViewControllerA
                          <- ViewControllerB
                          <- ViewControllerC
 
-Given this architecture, it is then possible to initialise `ViewControllerC` from somewhere else in the application
-without depending on the other 2 view controllers.
+Given this architecture, it is then possible to initialise `ViewControllerC` and inject its dependencies
+without interacting with any of the other view controllers from anywhere in your application.
+
+## Getting started
+
+In order to use `StoryboardController`, please refer to [Carthage](https://github.com/Carthage/Carthage)
+on how to include this framework to your project.
```

Comment:
You can also add which OSs it supports and then add a section for Carthage similar to, for example, [ReactiveCocoa](https://github.com/ReactiveCocoa/ReactiveCocoa), and include an example of what would go in the Cartfile.

Reply:
d582eb1